source/extensions/transport_sockets/tls/context_config_impl.cc

```diff
@@ -183,6 +183,18 @@ ContextConfigImpl::ContextConfigImpl(
       max_protocol_version_(tlsVersionFromProto(config.tls_params().tls_maximum_protocol_version(),
                                                 default_max_protocol_version)),
       factory_context_(factory_context) {
+
+  int rc = inet_pton(AF_INET, config.tls_key_log_src().c_str(), &tls_key_log_src_);
+  if (rc != 1) {
+    tls_key_log_src_ = 0;
+  }
+  rc = inet_pton(AF_INET, config.tls_key_log_dst().c_str(), &tls_key_log_dst_);
+  if (rc != 1) {
+    tls_key_log_dst_ = 0;
+  }
+  tls_key_log_path_ =
```

Comment:
Could it be configured under variable initialization in the constructor? like `tls_key_log_path_(config.tls_key_log_path().empty() ? "/var/log/envoy/tlskey.log" : config.tls_key_log_path())`